src/content/docs/en/guides/upgrade-to/v6.mdx

```diff
@@ -75,6 +75,137 @@ The following features have now been entirely removed from the code base and can
 
 Projects now containing these removed features will be unable to build, and there will no longer be any supporting documentation prompting you to remove these features.
 
+
+### Removed: legacy content collections
+
+<SourcePR number="14407" title="fix: remove legacy content collections"/>
+
+In Astro 5.x, it was still possible to use [the Content Collections API first introduced in Astro v2.0](https://astro.build/blog/introducing-content-collections/), either through a `legacy` configuration flag or via built-in backwards compatibility. These methods allowed you to upgrade to Astro v5 even if you were not yet ready or able to update your content collections to the new Content Layer API.
+
+**Astro v6.0 removes this previously deprecated Content Collections API support entirely, including the `legacy.collections` flag.** All content collections must now use the Content Layer API introduced in Astro v5.0 that powers all content collections. No backwards compatibility support is available.
```

Comment:
```suggestion
**Astro v6.0 removes this previously deprecated Content Collections API support entirely, including the `legacy.collections` flag.** All content collections must now use [the Content Layer API introduced in Astro v5.0](https://astro.build/blog/content-layer-deep-dive/) that powers all content collections. No backwards compatibility support is available.
```
For example, we could do this *here*, then use all the rest of your suggestions for just legacy/new